app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -35,6 +35,10 @@ def destroy
 
     private
 
+    def skip_sentry_exception_types
+      EXCEPTIONS_WITH_RECORD_NOT_FOUND
```

Comment:
isn't this just inherited? why list it if it is inherited from application controller?